js/storage.md

```diff
@@ -149,12 +152,13 @@ Import *Storage* from the aws-amplify library:
 import { Storage } from 'aws-amplify';
 ```
 
-If you use `aws-exports.js` file, Storage is already configured. To configure Storage manually,
+If you use `aws-exports.js` file, Storage is already configured when you call `Amplify.configure(awsmobile)`. To configure Storage manually,
 ```javascript
-Storage.configure({
-    bucket: //Your bucket ARN;
-    region: //Specify the region your bucket was created in;
-    identityPoolId: //Specify your identityPoolId for Auth and Unauth access to your bucket;
+Storage.configure(
```

Comment:
```
Storage.configure({
    AWSS3: {
        bucket: '', //Your bucket ARN
        region: '', //Specify the region your bucket was created in
     }
});
```

And I would add a comment that is required to configure Auth in order to S3 with credentials